spec/classes/foreman_service_spec.rb

```diff
@@ -14,10 +14,7 @@
       use_foreman_service: false,
       foreman_service: 'foreman',
       foreman_service_ensure: 'running',
-      foreman_service_enable: true,
-      jobs_service: 'dynflower',
-      jobs_service_ensure: 'stopped',
-      jobs_service_enable: false
+      foreman_service_enable: true
```

Comment:
Probably easier to set this to false so the tests pass. It's a bit complex to get the foreman variables and it's tested in other places as well. You can do it in `foreman_spec.rb` for example. This test is mostly there to test the apache logic that can be quite complex.